portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,22 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import java.util.ArrayList;
+import com.google.gson.Gson;
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  ArrayList<String> messages = new ArrayList<>();
```

Comment:
```suggestion
  List<String> messages = new ArrayList<>();
```
See effective java item 64 - refer to objects by their interfaces.

Reply:
Ok I'll do that.